deployment/updates/scripts/add_permissions/2022_05_11_update_kmc_and_ep_analytics_user_roles.php

```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * @package deployment
+ * @subpackage Quasar.roles_and_permissions
+ */
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionToRole.php';
+passthru("php $script 0 'Kmc analytics role' 'STUDIO_BASE' realrun");
```

Comment:
separate to two different scripts and replace the reference to them in the release notes